docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -125,30 +125,30 @@ The new accessible DOM structure will become compatible with Joy UI in the futur
 
 ## With a custom editing experience
 
-### Using an `Autocomplete`
+### Using an Autocomplete
 
 If your user can only select a value in a small list of available dates,
-you can replace the field with an `Autocomplete` listing those dates:
+you can replace the field with the `<Autocomplete />` from `@mui/material` to list those dates:
```

Comment:
I would style this as a proper noun rather than two backticked elements.

```suggestion
you can replace the field with Material UI Autocomplete to list those dates:
```

Reply:
I'm curious if @vladmoroz has any thoughts about formatting in this context.

Reply:
I'd say it like this personally...

> If your user can only select a value in a small list of available dates, you can replace the field with [Autocomplete](https://mui.com/material-ui/react-autocomplete/) to list those dates:

This sounds OK to me too:

> If your user can only select a value in a small list of available dates, you can replace the field with the Autocomplete component from `@mui/material` to list those dates:

- Proper noun either way
- I'd link out to Autocomplete because I need the API handy, and it saves us a reference to Material UI. Otherwise, my preference is explicit `@mui/material` instead of "Material UI" because people confuse MUI / Material UI / Material Design all the time.
- Either "with the Autocomplete component from..." or "with Autocomplete from...". Just "the Autocomplete" doesn't sound right to my ear.

Reply:
I updated the 3 sentences with a link to our `@mui/material` doc.

I just kept the "with the [Autocomplete]() component" because just "with [Autocomplete()" felt weird to me (same for "with [Button]().".